nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/PublicationContext.java

```diff
@@ -5,14 +5,15 @@
 
 /**
  * PublicationContext provides a common root object for contexts of type
- * {@link Partitive} and {@link BasicContext}.
+ * {@link BasicContext}.
  */
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 @JsonSubTypes({
     @JsonSubTypes.Type(name = "Book", value = Book.class),
     @JsonSubTypes.Type(name = "Report", value = Report.class),
     @JsonSubTypes.Type(name = "Degree", value = Degree.class),
-    @JsonSubTypes.Type(name = "Chapter", value = Chapter.class),
+    @JsonSubTypes.Type(name = "Anthology", value = Anthology.class),
```

Comment:
To ganger det samme feltet

Reply:
Må være en IntelliJ refactor ting. `Chapter` should remain there for migration purposes.

Reply:
Jepp, commit-and-run newb feil fra min side. Fikset